lib/cylc/scheduler.py

```diff
@@ -203,28 +203,25 @@ def __init__(self, is_restart, options, args):
 
     def start(self):
         """Start the server."""
+        self._check_port_file_exists(self.suite)
         self._print_blurb()
-        try:
```

Comment:
This `try...except` block is a bit pointless, as this method is caught by a similar block in the `main` function.